### ISSUE TITLE
Different icons for non-operational POIs

### ISSUE THREAD

**Issue by gaddman** [openchargemap/ocm-system#89]
The icon for a future charging station is the same as an operational station, and it is only the hover text which illustrates the difference. A different colour or icon to differentiate stations which aren't available would be helpful. Perhaps the set4 private (no entry marker) icons would work, or one of the level0 (grey) icons.

**Comment by webprofusion-chrisc** [comment]
Agreed. The icons are create in inkscape (SVG) and the latest version of the markers (for the new app) are at https://github.com/openchargemap/ocm-docs/blob/master/Branding/map-markers/map_markers.svg if you were feeling creative?

**Comment by webprofusion-chrisc** [comment]
For info, there is also a (very early) preview of the new app, which is a re-write:
http://map.openchargemap.io/ most functionality is still missing.

**Comment by gaddman** [comment]
I present 'the spanner', a stylised version of the gorgeous drop forged 12" adjustable wrench sitting beside me.
[markers.zip](https://github.com/openchargemap/ocm-system/files/100577/markers.zip)
I'd do a pull request but I'm no graphic designer and I'm not sure if I've added it to the icons correctly.

As an aside, there are emoji's for all these now. Not sure if that could work with a map icon though, or if you can control what colour then end up on different OSs, so may not be helpful. But just in case, here's a spare spanner: 🔧, no entry: ⛔, and charger: ⚡

**Comment by webprofusion-chrisc** [comment]
Nice work! Would you like to try submitting a pull request for map-markers.svg (fork it, apply your change then submit as a pull request)? Alternatively I can commit the file on your behalf, the difference is the level of recognition github gives you as a contributor.

I haven't exported these yet and don't quite know when I'll be integrating them but it looks good, that's the main thing.

Long term I'd like icons to be composited from a single marker shape, a color base applied and then any annotations applied on top (no entry, planned/under contruction, photos or checkins available etc). Ideally we'd do this from a single script rather than fiddling with the icons individually.
